ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-dependencies/js/utility.js

```diff
@@ -0,0 +1,29 @@
+const setUserCookie = function (cname, cvalue, exdays) {
```

Comment:
@rakeshkumar-verma @umeshv12 : Why has the cookie code written in hclecomm.dependencies js . This will be included in the page <head> tag.

refere: 
apps/hclecomm/components/structure/page/customheaderlibs.html

Reply:
This is common code required by multiple component or pages (like signin, signup and login button). It might be useful for other components in future which is dependent on cookie.
This is for code optimization.